### DISCUSSION TITLE
qskscrollarea

### DISCUSSION THREAD

**Discussion by hestrro** [uwerat/qskinny#500] [Q&A] [answered]
When qskscrollarea slides with the mouse, it will move a certain distance with inertia. Can this effect be set up with a function to turn it on and off?

**Comment by uwerat** [comment, marked as answer]
This distance is the QskPanGestureRecognizer::minDistance().

Unfortunately the gesture recognizer is a private member of QskScrollBox and is not accessible for application code - only the timeout can be modified ( QskScrollBox::setFlickRecognizerTimeout ). TODO ...
